.buildkite/pipeline.macos.yml

```diff
@@ -79,15 +79,23 @@ steps:
     - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/travis/upload_build_info.sh; fi }; trap cleanup EXIT
     - ./ci/travis/ci.sh test_cpp
 
-- label: ":mac: :apple: Small and Large"
+- label: ":mac: :apple: Small & Client"
```

Comment:
Can you revert the change for macos? We can add them back when we have more capacity.

Reply:
Or how about we wait until there is more capacity? Otherwise with a slow MacOS build step the overall build is still going to be slow..